bitbucket-scm-filter-aged-refs/src/test/java/org/jenkinsci/plugins/scm_filter/BitbucketAgedRefsTraitTest.java

```diff
@@ -1,30 +1,58 @@
 package org.jenkinsci.plugins.scm_filter;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
 
 import com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMSource;
-import java.io.IOException;
-import java.io.InputStream;
 import jenkins.model.Jenkins;
-import org.junit.jupiter.api.Test;
```

Comment:
Please don't revert from JUnit Jupiter to JUnit 4.

Reply:
Resolved.